src/Http/Controllers/ContactsController.php

```diff
@@ -2,18 +2,163 @@
 
 namespace Ourgarage\Contacts\Http\Controllers;
 
+use Ourgarage\Contacts\DTO\ImageResizeDTO;
 use App\Http\Controllers\Controller;
+use Ourgarage\Contacts\Services\UploadImageSaverService;
+use Ourgarage\Contacts\Http\Requests\ContactsFileUploadRequest;
+use Ourgarage\Contacts\Models\Contact;
+use Notifications;
+use File;
 
 class ContactsController extends Controller
 {
+    public function adminContactsIndex()
```

Comment:
If this controller is all about `admin` why not to put it into namespace Admin and remove this method prefixes? ;)